requirements-typecheck.txt

```diff
@@ -0,0 +1,3 @@
+mypy==0.971
+pyinstaller==5.8.0
+types-setuptools==67.2.0.1
```

Comment:
I'd rather not have another separate file for this if that's OK - let's add it to `test` as they're just packages that we need to test Eel. Type checking is just another type of test.

Reply:
Yes, totally happy to add it there. The thinking was to separate them as they're not needed for the other tests.

Reply:
Yeah, I understand the logic :) Don't think we need it though, so let's simplify a tad.

Reply:
I've integrated these into requirements-test.txt now, so we only have one.